### DISCUSSION TITLE
[JPA] 영속성 컨텍스트의 개념

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#136] [1. 개발 관련 글 모음]
[게시글 링크](https://amaran-th.github.io/Spring/[JPA]%20%EC%98%81%EC%86%8D%EC%84%B1%20%EA%B4%80%EB%A6%AC%EC%99%80%20%EC%98%81%EC%86%8D%EC%84%B1%20%EC%BB%A8%ED%85%8D%EC%8A%A4%ED%8A%B8/)

**Comment by shkisme** [comment]
김영한 JPA 책을 읽고 계시는군요~~👍
혹시 표나 도식화한 이미지들은 직접 하시는건가요??

**Reply by amaran-th** [reply]
네~~! 프로젝트에서 jpa를 쓰고있는데, 제가 jpa를 공부해본 적이 없어서 다른 팀원분들이 짜신 코드를 보고 눈치껏 사용하고 있었는데, 저번주에 바쁜 일정이 어느정도 마무리돼서 본격적으로 공부해보려고 합니다!
표는 css로 디자인 한거구, 이미지는 피그마로 직접 그렸습니다!! 피그마 쓰다보니까 이런 자료 만들 때 정말 편하더라구요ㅋㅋ

**Reply by shkisme** [reply]
네..?? css로 디자인이요...??? 피그마로 그린거라구요??? 넘 이뻐서 어떻게 만든건지 궁금했는데 대박신기하네요...... 저도 연습해보겠습니다...홀홀(동손)

jpa 쓸수록 느끼는거지만, 추상화 수준이 아주 높아서 깊은 지식 없이도 개발자가 쉽게 쓸 수 있는 것 같아요. 근데 그렇게 하면 한계가 있더군요... 저도 사용을 많이 하다보니 지식이 머릿속에만 남아있어서 이렇게 정리가 필요한 것 같네용... 

글이 잘 정돈되어 있어서 보기 편했습니다 👍👍👍

**Reply by amaran-th** [reply]
칭찬 감사합니다ㅎㅎ><
Jpa는...많이 사용해본 사람들도 파면 팔수록 어렵고 헷갈린다고 하더라구요
그만큼 기본기가 중요한것 같습니다...왜 이제야 시작했는지 쪼큼 후회가 되네요^-ㅠ
